hardware/I2C.cpp

```diff
@@ -167,35 +167,20 @@ bool I2C::WriteToHardware(const char *pdata, const unsigned char length)
 		_log.Log(LOG_NORM,"GPIO: id2 %d", pCmd->LIGHTING2.id2);
 		_log.Log(LOG_NORM,"GPIO: id3 %d", pCmd->LIGHTING2.id3);
 		_log.Log(LOG_NORM,"GPIO: id4 %d", pCmd->LIGHTING2.id4);
-		_log.Log(LOG_NORM,"GPIO: unitcode %d", pCmd->LIGHTING2.unitcode); // in DB columb "Unit" used for identify number switch on board
-		_log.Log(LOG_NORM,"GPIO: cmnd %d", pCmd->LIGHTING2.cmnd);
-		_log.Log(LOG_NORM,"GPIO: level %d", pCmd->LIGHTING2.level);
-		*/
 		unsigned char id1=pCmd->LIGHTING2.id1;
 		unsigned char id2=pCmd->LIGHTING2.id2;
 		unsigned char id3=pCmd->LIGHTING2.id3;
 		unsigned char id4=pCmd->LIGHTING2.id4;
 		int ID=(id1<<24)|(id2<<16)|(id3<<8)|id4;
-		unsigned char Unit = pCmd->LIGHTING2.unitcode;
-		unsigned char  value = pCmd->LIGHTING2.cmnd;
-		/*
-		_log.Log(LOG_NORM,"GPIO: ID %d", ID);
-		_log.Log(LOG_NORM,"GPIO: Unit %d", Unit);
-		_log.Log(LOG_NORM,"GPIO: new value %d", value);
-		*/
-		// check my serverd i2c_addr (form Unit I can get i2c_addres of switch)
-		unsigned char i2c_address=PCF8574_get_i2c_addr_from_Unit(Unit);
-		/*
-		_log.Log(LOG_NORM,"GPIO: my served i2c address %d", i2c_addr);
-		_log.Log(LOG_NORM,"GPIO: i2c address from Unit %d", i2c_address);
+		//_log.Log(LOG_NORM,"GPIO: ID %d", ID);
```

Comment:
OK I do. 

Last compile for apple had problem with connect: The remote end hung up unexpectedly
maybe some problem on git?

Is it
DeviceID = (HTYPE_RaspberryPCF8574 << 24) + (0 << 16) + (i2c_addr << 8) + pin_number
correct?